app/services/tools/base.py

```
@@ -345,6 +357,12 @@ def verifies_path(self, path: str) -> bool:
         # per-file inspection.
         return match_extension(path, self.verify_extensions) is not None
 
+    def scannable_path(self, path: str) -> bool:
```

Ruff (0.16.1)

[warning] Unused method argument: `path`

(ARG002)